src/sphinx/extensions/index.rst

```diff
@@ -17,7 +17,7 @@ Official Extensions
 Third Parties
 =============
 
-* `Gradle plugin <https://github.com/alphagov/gradle-gatling-plugin>`_
+* `Gradle plugin <https://github.com/lkishalmi/gradle-gatling-plugin>`_
```

Comment:
Hey, you're alive! Your github profile says Barcelona, are you still there? I think the one you owe is @ceeaspb, who's in London.